docs/4.2/installation.md

```diff
@@ -161,7 +161,7 @@ $ cd teleport
 # Make sure you have `zip` installed - the Makefile uses it
 $ make full
 # create the default data directory before running `teleport`
-$ sudo mkdir -p /var/lib/teleport
+$ sudo mkdir -p -m0700 /var/lib/teleport
```

Comment:
Good catch.

Reply:
We don't technically need to do this either - Teleport will create its own data directory when running for the first time.. With that said, it can only do this easily when running as `root` which should only be the case for nodes rather than auth/proxy.